pkg/images/enricher/enricher_impl_test.go

```diff
@@ -623,7 +634,9 @@ func TestNoMatchingRegistryIntegration(t *testing.T) {
 	img := &storage.Image{Id: "id", Name: &storage.ImageName{Registry: "reg"}}
 	results, err := enricherImpl.EnrichImage(emptyCtx, EnrichmentContext{}, img)
 	assert.Error(t, err)
-	expectedErrMsg := "image enrichment error: error getting metadata for image:  error: no matching image registries found: please add an image integration for reg"
+	expectedErrMsg := "image enrichment errors: [error getting metadata for image:  error: no matching image " +
```

Comment:
why did error change?

Reply:
We don't return on the first error occurred within the enrichment flow, but rather are creating an error list.
Previously, only `enrichWithMetadata` was accessing registry, now the `enrichWithSignature` step will also access registry information, thus the second error in the list since it's enabled now by default.